linux/inc/file_linux.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#ifndef FILE_LINUX_H
+#define FILE_LINUX_H
+
+#include "file_internal.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct FILE_LINUX_IO_TAG
+{
+    void* handle;
+    FILE_ASYNC_OPERATION type;
+    FILE_IO_DATA data;
+}FILE_LINUX_IO;

```

Comment:
>FILE_LINUX_IO [](start = 1, length = 13)

Looks like this is all internal stuff that can just go in the .c file, not sure this header is needed...